<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>user_manual</class>
 <widget class="QWidget" name="user_manual">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QTreeWidget" name="treeWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>420</width>
     <height>600</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>               user manual</string>
    </property>
   </column>
   <item>
    <property name="text">
     <string>Wifi</string>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>wifi connect</string>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
    </item>
    <item>
     <property name="text">
      <string>hot spot</string>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
    </item>
   </item>
   <item>
    <property name="text">
     <string>Bluetooth</string>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>BT scan</string>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
    </item>
    <item>
     <property name="text">
      <string>BT pair</string>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
    </item>
    <item>
     <property name="text">
      <string>BT connect</string>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
    </item>
   </item>
   <item>
    <property name="text">
     <string>Ethernet</string>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>show all ip connect</string>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
    </item>
    <item>
     <property name="text">
      <string>set dynamic ip </string>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
    </item>
    <item>
     <property name="text">
      <string>set static ip</string>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
    </item>
    <item>
     <property name="text">
      <string>network speed test</string>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
    </item>
   </item>
   <item>
    <property name="text">
     <string>RTC</string>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Brightness adjustment</string>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>brightness</string>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Sleep time</string>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
    </item>
   </item>
   <item>
    <property name="text">
     <string>GPIO</string>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>scan occupied gpio</string>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
    </item>
    <item>
     <property name="text">
      <string>batch operation</string>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
    </item>
   </item>
   <item>
    <property name="text">
     <string>Serial Port</string>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Voice Frequency</string>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>choose file</string>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
    </item>
    <item>
     <property name="text">
      <string>begin</string>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
    </item>
    <item>
     <property name="text">
      <string>play</string>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
    </item>
   </item>
   <item>
    <property name="text">
     <string>Touch Display</string>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Udev</string>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>udev</string>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
    </item>
   </item>
   <item>
    <property name="text">
     <string>All Interface Test</string>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>test</string>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
    </item>
    <item>
     <property name="text">
      <string>photo</string>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
    </item>
   </item>
   <item>
    <property name="text">
     <string>Key Test</string>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Chinese/English</string>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>2</y>
     <width>100</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="translation.qrc">
     <normaloff>:/t507_button_image/return.png</normaloff>:/t507_button_image/return.png</iconset>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>0</y>
     <width>604</width>
     <height>600</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>2</y>
     <width>150</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>expand all</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="translation.qrc"/>
 </resources>
 <connections/>
</ui>
